core/test_network/src/lib.rs

```diff
@@ -89,12 +89,43 @@ impl<G: GenesisNetworkTrait> TestGenesis for G {
             "wonderland".parse().expect("Valid"),
             get_key_pair().public_key().clone(),
         );
+        let rose_definition_id = <AssetDefinition as Identifiable>::Id::from_str("rose#wonderland")
+            .expect("valid names");
+        let alice_id =
+            <Account as Identifiable>::Id::from_str("alice@wonderland").expect("valid names");
+        let rose_id =
+            <Asset as Identifiable>::Id::new(rose_definition_id.clone(), alice_id.clone());
+
         genesis.transactions[0].isi.push(
             RegisterBox::new(AssetDefinition::quantity(
                 AssetDefinitionId::from_str("rose#wonderland").expect("valid names"),
             ))
             .into(),
         );
+        genesis.transactions[0].isi.push(
+            GrantBox::new(
+                PermissionToken::new(
+                    Name::from_str("can_mint_user_asset_definitions").expect("valid names"),
+                )
+                .with_params([(
+                    Name::from_str("asset_definition_id").expect("valid names"),
+                    IdBox::AssetDefinitionId(rose_definition_id.clone()).into(),
+                )]),
+                alice_id.clone(),
+            )
+            .into(),
+        );
+        genesis.transactions[0].isi.push(
+            GrantBox::new(
+                PermissionToken::new(Name::from_str("can_burn_user_assets").expect("valid names"))
+                    .with_params([(
+                        Name::from_str("asset_definition_id").expect("valid names"),
```

Comment:
```suggestion
                        Name::from_str("asset_id").expect("valid names"),
```

Reply:
Maybe we should use specialized tokens e.g. `CanBurnUserAssets` here

Reply:
Hm... I didn't know we have this possibility... `declare_token!` should be documented